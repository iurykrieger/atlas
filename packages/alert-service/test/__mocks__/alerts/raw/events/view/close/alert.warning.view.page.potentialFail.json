{
	"action": "Close",
	"alert": {
		"alertId": "80ab888f-487f-480a-9148-e958f25b75d3-1541857819191",
		"message": "[DC][prophetheus] page_views {\"apiKey\": \"chillibeans\"}",
		"tags": [
			"dc-prophetheus",
			"atd",
			"int"
		],
		"tinyId": "7330",
		"entity": "",
		"alias": "[DC][prophetheus] page_views {\"apiKey\": \"chillibeans\"}",
		"createdAt": 1535461323437,
		"updatedAt": 1535461323437000000,
		"username": "Alert API",
		"userId": "",
		"description": "[DC][prophetheus] page_views {\"apiKey\": \"chillibeans\"}",
		"teams": [
			"afa919f9-5c24-4b55-b60e-2548bdda7abf",
			"e4e9ae6d-cec6-4cd1-9fb1-a4c4a710fc83"
		],
		"priority": "P3",
		"source": "dc-ops"
	},
	"source": {
		"type": "API"
	},
	"integrationName": "Webhook-TAM",
	"integrationId": "75d1e86f-475f-484f-8b6f-0a3a276de9b5",
	"integrationType": "Webhook"
}
