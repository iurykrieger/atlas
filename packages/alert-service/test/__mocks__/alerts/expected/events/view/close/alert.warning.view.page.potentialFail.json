{
	"id": "[DC][prophetheus] page_views {\"apiKey\": \"chillibeans\"}",
	"status": "closed",
	"apiKey": "chillibeans",
	"type": "alert.warning.view.page.potentialFail",
	"data": {
		"alertId": "80ab888f-487f-480a-9148-e958f25b75d3-1541857819191",
		"message": "[DC][prophetheus] page_views {\"apiKey\": \"chillibeans\"}",
		"tags": [
			"dc-prophetheus",
			"atd",
			"int"
		],
		"tinyId": "7330",
		"entity": "",
		"createdAt": 1535461323437,
		"updatedAt": 1535461323437000000,
		"username": "Alert API",
		"userId": "",
		"description": "[DC][prophetheus] page_views {\"apiKey\": \"chillibeans\"}",
		"teams": [
			"afa919f9-5c24-4b55-b60e-2548bdda7abf",
			"e4e9ae6d-cec6-4cd1-9fb1-a4c4a710fc83"
		],
		"priority": "P3",
		"source": "dc-ops"
	}
}
